plonk-core/src/proof_system/prover.rs

```diff
@@ -78,10 +80,9 @@ where
         &mut self.cs
     }
 
-    /// Returns the number of gates in the circuit thet the `Prover` actually
-    /// stores inside.
-    pub fn circuit_size(&self) -> usize {
-        self.cs.circuit_size()
+    /// Returns the smallest power of two needed for the curcuit
+    pub fn circuit_bound(&self) -> usize {
+        self.cs.circuit_bound()
```

Comment:
I think these functions do not collide. One expresses the required bound while the other returns the current number of gates.

Reply:
`circuit_size` was removed mainly to avoid confusion and prevent misuse, as the value that is normally needed is the smallest power of two that can fit all the circuit (including gates and lookup table).
The number of gates is needed only for a couple of cases and it can be accessed through the standard composer directly.

Reply:
Ohh ye it's `n`.
Fair enough!